ReactiveCocoa/Swift/Property.swift

```diff
@@ -138,46 +136,22 @@ public func <~ <T>(property: MutableProperty<T>, producer: SignalProducer<T, NoE
 	let propertyDisposable = property.producer.start(completed: {
```

Comment:
Ah, yeah, just save the disposable given to `startWithSignal`. I know it's a little unclean, but it's totally valid—the only reason that method doesn't itself return a `Disposable` is because I thought that'd confuse its purpose too much.